src/main/java/com/powsybl/openloadflow/network/AbstractLfBranch.java

```diff
@@ -117,17 +114,19 @@ public LfBus getBus2() {
     }
 
     protected List<LfLimit> getLimits1(CurrentLimits currentLimits) {
```

Comment:
Shouldn't we deprecate those 2 getLimits?

Reply:
What do you mean by deprecate? These methods have to be removed.

Reply:
If we want this change to not be a breaking change, I think we must keep them as deprecated for a while.

Reply:
A breaking change for who? This is the internal network of the open loadlfow engine, nobody use that `LfNetwork` except the engine.

Reply:
I you say it is safe to break, I'll remove them.